### PR TITLE
Bugfix: Our InviteStatus now has a status called EXPIRED, 

### PR DESCRIPTION
**Pull request by KevinFHartmann**
which we return whenever the invitation was PENDING and is expired.

@andricDu Adding you because we need two people to review PRs when they have changes to the protobufs file (in this case, the new enum value "EXPIRED").

### CODE REVIEW

File: src/main/proto/ProgramService.proto [hunk 1]
```diff
@@ -91,6 +91,7 @@ enum InviteStatus {
   REVOKED = 0;
   PENDING = 1;
   ACCEPTED = 2;
+  EXPIRED  = 3;
```
Comment (andricDu) [inline src/main/proto/ProgramService.proto]: @joneubank heads up on this one

File: src/main/proto/ProgramService.proto [hunk 2]
```diff
@@ -156,7 +157,7 @@ message ListUserRequest {
 }
 
 message ListUserResponse {
-  repeated Invitation invitations = 1;
+  repeated UserStatus invitations = 1;
```
Comment (joneubank) [inline src/main/proto/ProgramService.proto]: why not `repeated UserStatus users = 1;` ?

File: src/main/proto/ProgramService.proto [hunk 3]
```diff
@@ -28,18 +28,18 @@ option objc_class_prefix = "PSP";
...
-  rpc InviteUser(InviteUserRequest) returns (InviteUserResponse);
-  rpc JoinProgram(JoinProgramRequest) returns (JoinProgramResponse);
+  rpc InviteUser (InviteUserRequest) returns (InviteUserResponse);
+  rpc JoinProgram (JoinProgramRequest) returns (JoinProgramResponse);
 
-  rpc RemoveUser(RemoveUserRequest) returns (RemoveUserResponse);
-  rpc ListUser(ListUserRequest) returns (ListUserResponse);
-  rpc UpdateUser(UpdateUserRequest) returns (google.protobuf.Empty);
+  rpc RemoveUser (RemoveUserRequest) returns (RemoveUserResponse);
+  rpc ListUsers (ListUserRequest) returns (ListUserResponse);
```
Comment (joneubank) [inline src/main/proto/ProgramService.proto]: 👍

File: src/main/proto/ProgramService.proto (hunk 2 above)
Comment (KevinFHartmann) [reply src/main/proto/ProgramService.proto]: Because I missed that. :-( Thanks for catching it!

**Pull request merged**